src/main/java/mate/academy/spring/controllers/RentController.java

```diff
@@ -0,0 +1,39 @@
+package mate.academy.spring.controllers;
+
+import mate.academy.spring.service.BookService;
+import mate.academy.spring.service.RentService;
+import mate.academy.spring.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.ModelMap;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@Controller
+@RequestMapping("/rent")
+public class RentController {
+    @Autowired
+    UserService userService;
+
+    @Autowired
+    BookService bookService;
```

Comment:
```suggestion
    BookService bookService;
```
@Sasha192 По-перше а ти використовуєш bookService в цьому класу? А по-друге: чому без модифікатора private&